budgetportal/views.py

```diff
@@ -11,18 +11,18 @@
 
 from budgetportal.csv_gen import generate_csv_response
 from budgetportal.openspending import PAGE_SIZE
-import nav_bar
-from datasets import Category, Dataset
+from .nav_bar import get_items
```

Comment:
this is a very vague method name when it's not qualified with `nav_bar.` - please leave it as is